src/components/date-range/date-range.test.js

```diff
@@ -506,204 +506,193 @@ context("Test for DateRange component", () => {
+    it.each(testData)(
+      "should check accessibility with the endInfo as string renders as %s",
+      (endInfo) => {
+        CypressMountWithProviders(<DateRangeCustom endInfo={endInfo} />);
 
-          cy.checkAccessibility();
-        }
-      );
+        cy.checkAccessibility();
+      }
+    );
 
-      it.each(testData)(
-        "should check accessibility with the validationOnLabel with error state",
-        (error) => {
-          CypressMountWithProviders(
-            <DateRangeCustom
-              startError={error}
-              endError={error}
-              validationOnLabel
-            />
-          );
+    it.each(testData)(
+      "should check accessibility with the validationOnLabel with error state",
+      (error) => {
+        CypressMountWithProviders(
+          <DateRangeCustom
+            startError={error}
+            endError={error}
+            validationOnLabel
+          />
+        );
 
-          cy.checkAccessibility();
-        }
-      );
+        cy.checkAccessibility();
+      }
+    );
 
-      it.each(testData)(
-        "should check accessibility with the validationOnLabel with warning state",
-        (warning) => {
-          CypressMountWithProviders(
-            <DateRangeCustom
-              startWarning={warning}
-              endWarning={warning}
-              validationOnLabel
-            />
-          );
+    it.each(testData)(
+      "should check accessibility with the validationOnLabel with warning state",
+      (warning) => {
+        CypressMountWithProviders(
+          <DateRangeCustom
+            startWarning={warning}
+            endWarning={warning}
+            validationOnLabel
+          />
+        );
 
-          cy.checkAccessibility();
-        }
-      );
+        cy.checkAccessibility();
+      }
+    );
 
-      it.each(testData)(
-        "should check accessibility with the validationOnLabel with info state",
-        (info) => {
-          CypressMountWithProviders(
-            <DateRangeCustom
-              startInfo={info}
-              endInfo={info}
-              validationOnLabel
-            />
-          );
+    it.each(testData)(
+      "should check accessibility with the validationOnLabel with info state",
+      (info) => {
+        CypressMountWithProviders(
+          <DateRangeCustom startInfo={info} endInfo={info} validationOnLabel />
+        );
 
-          cy.checkAccessibility();
-        }
-      );
+        cy.checkAccessibility();
+      }
+    );
 
-      it.each([["top"], ["bottom"], ["left"], ["right"]])(
-        "should check accessibility with the tooltipPosition is set to %s",
-        (position) => {
-          CypressMountWithProviders(
-            <DateRangeCustom
-              m={9}
-              tooltipPosition={position}
-              startError={testCypressText}
-            />
-          );
+    it.each([["top"], ["bottom"], ["left"], ["right"]])(
```

Comment:
This can also be written as this :)
```suggestion
    it.each(["top", "bottom", "left", "right"])(
```

Reply:
changed